python/django/Makefile

```diff
@@ -4,4 +4,4 @@ start:
 
 deps:
 	pip3 install --upgrade setuptools
-	pip3 install django==3.0.9 django-cockroachdb==3.0.1 psycopg2
+	pip3 install django==3.1.* django-cockroachdb==3.1.* psycopg2
```

Comment:
could we pin them to specific versions -- we've had issues in the past where the test started picking up the latest version but that broke the test since some config/setup needed changes. and when this test breaks, it blocks development on cockroachdb/cockroach so PRs can't get merged

Reply:
Got it. Fixed the version to 3.1.4.